graylog2-web-interface/src/views/components/contexts/WidgetFocusProvider.tsx

```diff
@@ -84,10 +84,7 @@ const useSyncStateWithQueryParams = ({ focusedWidget, focusUriParams, setFocused
       setFocusedWidget(nextFocusedWidget);
       const filter = nextFocusedWidget?.id ? [nextFocusedWidget.id] : null;
       SearchActions.setWidgetsToSearch(filter);
-
-      if (focusedWidget?.editing && filter === null) {
-        SearchActions.executeWithCurrentState();
-      }
+      SearchActions.executeWithCurrentState();
```

Comment:
As discussed:

With this change we are currently executing the search twice when opening the search page.

We are now also executing the search when we focus / edit a widget, aas a result we loos the scroll position in the log view widget or the current page of the message table when focusing a widget.

One possible solution could be changing `focusedWidget?.editing` to `focusedWidget?.focusing`.